udata_gouvfr/harvesters/maaf.py

```diff
@@ -111,7 +108,7 @@
 
 
 def extract(element):
-    lst = filter(lambda r: isinstance(r[0], basestring), map(dictize, element))
+    lst = filter(lambda r: isinstance(r[0], str), map(dictize, element))
```

Comment:
List comprehension ?

Reply:
👍

Reply:
PS: this is not tested :-( I hope I won't break anything.